index.html

```diff
@@ -69,5 +77,296 @@
       </ul>
       <div class="some"></div>
     </div>
+    <div class="recent-works">
+      <div class="title">
+        <h1 class="head-title">My Recent Works</h1>

```

Comment:
- [ ] Kindly change this `h1` tag to any of the `hx (h2-h6)` tags. It is the convention and best practice to use only one `h1` tag on a webpage. Kindly change it 👍